pkg/test/framework/components/istio/config.go

```diff
@@ -153,6 +153,9 @@ type Config struct {
 
 	// EnableCNI indicates the test should have CNI enabled.
 	EnableCNI bool
+
+	// Custom External Control Plane cluster values
+	ExternalControlPlaneClusterValues string
```

Comment:
nit: maybe move this up to line 117, so it's close to `ControlPlaneValues` and rename to `ExternalControlPlaneValues` for better name consistency.